x-pack/plugin/inference/src/test/java/org/elasticsearch/xpack/inference/mapper/SemanticInferenceMetadataFieldsMapperTests.java

```diff
@@ -55,6 +55,23 @@ public void testIsEnabled() {
         assertTrue(InferenceMetadataFieldsMapper.isEnabled(settings));
     }
 
+    public void testIsEnabledByDefault() {
+        // Check index version SOURCE_MAPPER_MODE_ATTRIBUTE_NOOP because that index version was added in the same serverless promotion
+        // where the new format was enabled by default
+        var settings = Settings.builder()
+            .put(
+                IndexMetadata.SETTING_INDEX_VERSION_CREATED.getKey(),
+                IndexVersionUtils.getPreviousVersion(IndexVersions.SOURCE_MAPPER_MODE_ATTRIBUTE_NOOP)
```

Comment:
Here, you could reference the index version variable I suggested above, and it would be cleaner - and there would only be the need for one comment.